test/checkoutput/README.md

```diff
@@ -0,0 +1,45 @@
+# checkoutput tool
+
+The checkoutput tool parses a text or markdown that has a command and the expected output sequence, runs the command and checks the output of the command against the expected output. 
```

Comment:
markdown is spelled Markdown (capital letter).

Reply:
Fixed.